### PR TITLE
wrapAll: change description of function argument

### PR DESCRIPTION
**Pull request by arthurvr**
For the v3 branch.

Fixes gh-605

### CODE REVIEW

**Comment by joshyop** [comment]
@arthurvr here are some dicks for you fag:

File: entries/wrapAll.xml [hunk 1]
```diff
@@ -14,8 +14,7 @@
   <signature>
     <added>1.4</added>
     <argument name="function" type="Function">
-      <desc>A callback function returning the HTML content or jQuery object to wrap around the matched elements. Receives the index position of the element in the set as an argument. Within the function, <code>this</code> refers to the current element in the set.</desc>
-      <argument name="index" type="Integer" />
+      <desc>A callback function returning the HTML content or jQuery object to wrap around all the matched elements. Within the function, <code>this</code> refers to the first element in the set. <strong>Prior to jQuery 3.0</strong>, the callback got called for every element in the set and received the index position of the element in the set as an argument.</desc>
```
Comment (dmethvin) [inline entries/wrapAll.xml]: How about changing "the callback got called" to "the callback was incorrectly called" because it was the wrong behavior.

Comment (arthurvr) [inline entries/wrapAll.xml]: That sounds like a good idea :+1: 

@dmethvin Added. LGTY?

**Comment by dmethvin** [comment]
:+1:

**Comment by arthurvr** [comment]
Closing as this is merged in the v3 branch.

**Pull request closed without merging**